### PR TITLE
BotkindInc.AllwaySync version 21.0.9

### PR DESCRIPTION
**Pull request by techflashYT**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] Have you validated your manifest locally with `winget validate --manifest <path>`? 
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.0 schema](https://github.com/microsoft/winget-cli/blob/master/doc/ManifestSpecv1.0.md)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/16422)

### CODE REVIEW

File: manifests/b/BotkindInc/AllwaySync/21.0.9/BotkindInc.AllwaySync.yaml [hunk 1]
```diff
@@ -0,0 +1,7 @@
+# Created using wingetcreate
+
+PackageIdentifier: BotkindInc.AllwaySync
```
Comment (ItzLevvie) [inline manifests/b/BotkindInc/AllwaySync/21.0.9/BotkindInc.AllwaySync.yaml:3]:
```suggestion
PackageIdentifier: Botkind.AllwaySync
```

File: manifests/b/BotkindInc/AllwaySync/21.0.9/BotkindInc.AllwaySync.locale.en-US.yaml [hunk 2]
```diff
@@ -0,0 +1,11 @@
+# Created using wingetcreate
+
+PackageIdentifier: BotkindInc.AllwaySync
```
Comment (ItzLevvie) [inline manifests/b/BotkindInc/AllwaySync/21.0.9/BotkindInc.AllwaySync.locale.en-US.yaml:3]:
```suggestion
PackageIdentifier: Botkind.AllwaySync
```

File: manifests/b/BotkindInc/AllwaySync/21.0.9/BotkindInc.AllwaySync.installer.yaml [hunk 3]
```diff
@@ -0,0 +1,11 @@
+# Created using wingetcreate
+
+PackageIdentifier: BotkindInc.AllwaySync
```
Comment (ItzLevvie) [inline manifests/b/BotkindInc/AllwaySync/21.0.9/BotkindInc.AllwaySync.installer.yaml:3]:
```suggestion
PackageIdentifier: Botkind.AllwaySync
```

**Review by ItzLevvie — CHANGES_REQUESTED** [review]
Other than the changes I've made, everything else looks good to me:
image

**Comment by sanikdah** [comment]
sorry about that, github notifications are weird, didn't know that you suggested changes.

**Comment by sanikdah** [comment]
ugh, i was trying to play mario maker 2 -_-

**Comment by sanikdah** [comment]
hmm, well that's odd.

**Review by OfficialEsco — APPROVED** [review]

**Pull request merged**